packages/cli-lib/personalAccessKey.js

```diff
@@ -145,13 +146,33 @@ const updateConfigWithPersonalAccessKey = async (configData, makeDefault) => {
   }
   const { portalId, accessToken, expiresAt } = token;
 
+  let hubInfo;
+  try {
+    hubInfo = await fetchHubData(accessToken, portalId, accountEnv);
+  } catch (err) {
+    // Ignore error, returns 404 if account is not a sandbox
+  }
+
+  let sandboxType = null;
+  let parentHubId = null;
+  if (hubInfo) {
+    if (hubInfo.type !== undefined) {
+      sandboxType = hubInfo.type === null ? 'STANDARD' : hubInfo.type;
+    }
+    if (hubInfo.parentHubId) {
+      parentHubId = hubInfo.parentHubId;
+    }
+  }
+
   const updatedConfig = updateAccountConfig({
     portalId,
     personalAccessKey,
     name,
     environment: getValidEnv(accountEnv, true),
     authType: PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
     tokenInfo: { accessToken, expiresAt },
+    sandboxType,
```

Comment:
Had toyed around with that idea, but I ended up going with the `sandboxType` route since we may open up the CLI to standard sandboxes in addition to development sandboxes down the road. `hubType` is kind of limited to "customer" or "sandbox", and I didn't want to add too many properties in here